test/src/Cases.zig

```diff
@@ -705,6 +725,10 @@ pub fn lowerToBuildSteps(
 
         switch (update.case) {
             .Compile => {
+                // Force the binary to be emitted if requested.
+                if (case.emit_bin) {
+                    _ = artifact.getEmittedBin();
+                }
```

Comment:
This feels very hacky; definitely open to better ideas.

Reply:
Well, it seems `std.Build` itself uses this pattern in some places, so at least there's precedent... though it still seems like there ought to be a more user-friendly way to tell the build system that codegen is required.